src/main/webapp/user_profile.html

```diff
@@ -47,6 +47,13 @@ <h2>User Checklist:</h2>
           <button class="complete" type="button" onclick="goToTask(4)">Task 4</button>
         </li>
       </ul>
+      <form method="GET" action="/user">
+        <button type="submit" values="users" id="button" onClick="loadAllUsers()">Get Users</button>
```

Comment:
still the lint error: src/main/webapp/user_profile.html: line 51, col 58, attribute names must match the format

Reply:
`onclick` is supposed to be all lowercase?